doc/DEVELOPMENT.rst

```diff
@@ -216,3 +216,39 @@ Automatic
 
 To do the above steps automatically, run ``scripts/macos-deps-install.sh``.
 
+Graphical Assets
+----------------
+
+Taisei's GFX library is made up of a collection of sprites, shaders, and a few
+3D models. The 3D models are exclusively used for background scenery, while the
+sprites are used in everything from UI elements, to character portraits, to the
+danmaku bullets themselves.
+
+To modify the 3D models, you'll need `Blender <https://blender.org>`__, which is
+free and open source. Look for tutorials on YouTube for how to use it. The
+models themselves are located in ``resources/00-taisei.pkgdir/models``.
+
+Music and sound effects are located in ``resources/00-taisei.pkgdir/sfx``.
+
+For sprites, any image editor will do. Sprites are locaed in ``atlas``, in
+``.png`` format. However, to have sprites properly appear in Taisei, you'll
```

Comment:
Lossless `.webp` should generally be preferred for all new images (they are significantly smaller); the PNGs in the `atlas/` dir are mostly remains from before the webp switch. We have a script to automatically optimize images, `scripts/optimize-img.sh` and it's whole-directory wrapper, `scripts/optimize-all-img.sh` (these probably should be rewritten in Python eventually).

The script will optimize PNGs with [Leanify](https://github.com/JayXon/Leanify), convert them to WEBPs (with the official `cwebp` tool), and keep whichever is smaller (usually the webp, but there are rare exceptions). It'll also try to losslessly recompress WEBPs. Generally it should be ran on all newly added or edited images, except atlases (the atlas generator optimizes them already).